packages/firebase_analytics/firebase_analytics_platform_interface/pubspec.yaml

```diff
@@ -11,6 +11,7 @@ environment:
   flutter: '>=1.9.1+hotfix.5'
 
 dependencies:
+  collection: ^1.15.0
```

Comment:
IS this right, on the Analytics package?

Reply:
Oopsy